actors/builtin/power/power_state.go

```diff
@@ -136,10 +136,11 @@ func (st *State) setMinerBalance(store adt.Store, miner addr.Address, amount abi
 func (st *State) addMinerBalance(store adt.Store, miner addr.Address, amount abi.TokenAmount) error {
 	Assert(amount.GreaterThanEqual(big.Zero()))
 	table := adt.AsBalanceTable(store, st.EscrowTable)
-	if table.Add(miner, amount) == nil {
+	err := table.Add(miner, amount)
+	if err == nil {
```

Comment:
Would you mind rewriting this as:

```
if err := table.Add(miner, amount); err != nil {
	return err
}
st.EscrowTable = table.Root()
return nil
```

... which is more canonical Go. Someone (probably me) trying to be tricky caused the bug in the first place.

(I know Go isn't your primary language, thanks for the fixes)